12/umbraco-cms/tutorials/members-registration-and-login.md

```diff
@@ -97,7 +97,7 @@ To restrict access you need to have made a member group, and a page containing t
 
 Congratulations! With all of that setup, the page you chose will redirect the user to the Login page if they are not logged in, and if they are logged in - they will be able to see the page's contents.
 
-In the navigation, you will be able to tell the content node is restricted - the small red icon that is added on top of the document type icon signifies that.
+In the navigation, you will be able to tell the content node is restricted - the small red icon that is added on top of the Document Type icon signifies that.
```

Comment:
one of our vale rules is to have sentences less than 25 words, so in this case for this sentence to fit into the 25 words it can be split like this: 
```suggestion
In the navigation, you will be able to tell that the content node is restricted. This small red icon that is added on top of the Document Type icon signifies that.
```

Reply:
In order to keep the documentation consistent, can you please make this same change to version 10 and 11 of CMS as well? 😊

Reply:
Sure. I'll add the changes.